plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -584,18 +584,36 @@ public void defFileBasedDisk(String filePath, String diskLabel, DiskBus bus, Dis
 
         /* skip iso label */
         private String getDevLabel(int devId, DiskBus bus) {
```

Comment:
Would be great to have unit tests for either `getDevLabel(int devId, DiskBus bus)` or `getDevLabelSuffix(int deviceIndex)`, especially to test for the expected results when `devId` (or `deviceIndex`) are high enough to return a double-letter device label suffix.